src/providers/twitch/TwitchChannel.cpp

```diff
@@ -1203,7 +1230,8 @@ void TwitchChannel::loadRecentMessagesReconnect()
                 return;
 
             tc->loadingRecentMessages_.clear();
-        });
+        },
+        limit, this->disconnectedAt_, now, true);
```

Comment:
```suggestion
        limit, /*after=*/ this->disconnectedAt_, /*before=*/ now, /*jitter=*/ true);
```

Reply:
For anyone opening the code in an IDE, it will be clear what it does as clangd can highlight which param is named what, so I'd prefer to not use comments like this